index.html

```diff
@@ -1,120 +1,90 @@
-	<li>Type <code>sbpm install &lt;package> /upgrade</code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
-	<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
-        </ol>
+	<!-- Installation Section -->
+	<div class="jumbotron">
+	  <h1 class="display-4">Installation</h1>
+	  <p class="lead">Download the setup file below, and double-click to run it.</p>
+	  <p>There will be two new folders: <code>extension</code> and <code>logs</code>, and five new files: <code>sbpm.bat</code>, <code>install.bat</code>, <code>uninstall.bat</code>, <code>update.cmd</code>, and <code>reinstall.bat</code>.</p>
+	  <p class="lead">
+	    <a class="btn btn-primary btn-lg" href="https://smelt-modding.github.io/smelt/setup.bat" role="button">Download</a>
+	  </p>
 	</div>
-	
-	
-        <div class="boxes">
-        <h3 style="color:red;">To uninstall SBPM completely:</h3>
-        <ol>
-        <li>Double-click <code>uninstall.bat</code></li>
-	<li>You're done! A log will be available at <code>logs\uninstall.log</code>.</li>
-	<li>:(</li>
-	</ol>
-	</div>
-					
-        <div class="boxes">
-        <h3>To uninstall a command installed with SBPM:</h3>
-	<ol>
-	<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
-	<li>Type <code>sbpm uninstall &lt;package></code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
-	<li>You're done! A log will be available at <code>logs\uninstall.log</code>.</li>
-	</ol>
+
+	<!-- Usage Section -->
+	<div class="jumbotron">
+	  <h1 class="display-4">Usage</h1>
+		<br/>
+		<h2>To install commands pre-downloaded from other sources:</h2>
+	  <p class="lead">Move the downloaded commands into the extensions folder, Double-click <code>install.bat</code>, and you're done! A log will be available at <code>logs\install.log</code>.</p>
+    <br/>
+		<h2>To install commands available here:</h2>
+		<p class="lead">Open Command Prompt (Win+R, type cmd and hit Enter/Win+X, C) then type sbpm install <code>_package_</code> (replace <code>_package_</code> with the name of the desired package.) and you're done! A log will be available at <code>logs\install.log</code>.</p>
+		<br/>
+		<h2>To upgrade commands pre-downloaded from other sources:</h2>
+		<p class="lead">Delete the previous versions from the <code>extension</code> folder and Move the new versions into the <code>extension</code> folder. Now just double-click <code>reinstall.bat</code>, and you're done! Logs will be available at <code>logs\uninstall.log</code> and <code>logs\install.log</code>.</p>
+		<br/>
+		<h2>To upgrade commands available here:</h2>
+    <p class="lead">Open Command Prompt (Win+R, type cmd and hit Enter/Win+X, C) and type sbpm install <code>_package_/upgrade</code> (replace <code>_package_</code> with the name of the desired package). You're done! A log will be available at <code>logs\install.log</code>.</p>
```

Comment:
Replace `sbpm install <code>_package_/upgrade</code>` with `<code>sbpm install _package_ /upgrade</code>`.
1. `sbpm install` is part of the command...
2. There is a space between `_package_` and `/upgrade` - so `_package_ /upgrade` instead of `_package_/upgrade`.

Also, you might want to make a note of `/U` being a shorter alternative to `/upgrade` - both do the same thing, but if you know the former, it's quicker.